js/ooyala_ssai.js

```diff
@@ -326,6 +326,8 @@ OO.Ads.manager(function(_, $)
      */
     this.pauseAd = function(ad)
     {
+      //Removing the ad timeout since ad was paused
+      clearTimeout(this.adIdDictionary[ad.ad.data.id].adTimer);
```

Comment:
Can we add null checks for the following?
ad
ad.ad
ad.ad.data
this.adIdDictionary[ad.ad.data.id]?

Reply:
Done